plugins/auth-backend/src/service/router.ts

```diff
@@ -76,9 +76,12 @@ export async function createRouter(
 
   const appUrl = config.getString('app.baseUrl');
   const authUrl = await discovery.getExternalBaseUrl('auth');
-
+  const tokenExpirationSeconds = config.getOptionalNumber(
```

Comment:
You can call it `backstageTokenExpiration` and make it a `HumanDuration`.

You also have to add it to config.d.ts, probably with frontend visibility.

https://backstage.io/docs/conf/defining

https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/config.d.ts#L184

https://github.com/backstage/backstage/blob/ddbed9a6dc34ffba78297c1788c73df05bcac5e0/plugins/catalog-backend/src/service/CatalogBuilder.ts#L790C20-L790C20